distage/distage-framework-docker/src/main/scala/izumi/distage/docker/DockerContainer.scala

```diff
@@ -15,15 +15,17 @@ final case class DockerContainer[Tag](
   connectivity: ContainerConnectivity,
   containerConfig: Docker.ContainerConfig[Tag],
   clientConfig: ClientConfig,
-  availablePorts: VerifiedContainerConnectivity,
+  availablePorts: Option[AvailablePorts],
```

Comment:
This makes the API worse and doesn't actually increase safety. (see comment below) Could you please revert?